scene/gui/progress_bar.cpp

```diff
@@ -50,9 +50,58 @@ Size2 ProgressBar::get_minimum_size() const {
 
 void ProgressBar::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_INTERNAL_PHYSICS_PROCESS: {
+			if (is_visible_in_tree() && (!Engine::get_singleton()->is_editor_hint() || editor_preview_indeterminate)) {
```

Comment:
It would be best if the processing wasn't enabled at all. That would make the editor part of this check unnecessary.